### PR TITLE
Add helm directory

### PR DESCRIPTION
**Pull request by franktommasi**

### CODE REVIEW

**Comment by franktommasi** [comment]
The list of maintainers needs to be updated with the current maintainers.

File: helm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: Octarine k8s cluster risk assessment tool
+engine: gotpl
+maintainers:
+name: kube-scan
+sources:
+- https://github.com/octarinesec/kube-scan
+version: 0.1
+appVersion: 20.0
```
Comment (BenRub) [inline helm/Chart.yaml]: 20.2
We'll add this to next version.

File: helm/templates/cr.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (BenRub) [inline helm/templates/cr.yaml]: v1

File: helm/templates/crb.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (BenRub) [inline helm/templates/crb.yaml]: v1

File: helm/templates/service.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "name" . }}
+  labels:
+    app: {{ template "name" . }}
+spec:
+  ports:
+    - name: kube-scan-ui
```
Comment (BenRub) [inline helm/templates/service.yaml]: ports:
    - name: {{ template "name" . }}-ui

File: helm/values.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+##Values to be provided to the deploment
+deployment:
+  labels:
+  ##Any labels required for the pods
+
+##Image specs
+image:
+  ui:
+    repository: octarinesec/kubescan-scanner-ui
+    tag: "20.0"
```
Comment (BenRub) [inline helm/values.yaml]: 20.2
We'll add this to next version.
If you need to run tests, you can type in your local clone "20.1.1"
Same for the second image.

File: helm/templates/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "name" . }}
+  template:
+    metadata:
+      labels:
+{{ toYaml .Values.deployment.labels | indent 8 }}
```
Comment (BenRub) [inline helm/templates/deployment.yaml]: {{- if .Values.deployment.labels }}
{{ toYaml .Values.deployment.labels | indent 8 }}
{{- end }}

**Comment by BenRub** [comment]
Hi @franktommasi 
How are you?
Did you get a chance to go over the changes request?

Thanks, Ben.

**Comment by franktommasi** [comment]
I'll have to redo this to include all the recent changes. I'll include your comments above as well. :)

**Comment by BenRub** [comment]
Thanks!

**Pull request closed without merging**